<launch>
  <!-- assumes ds4drv is running -->
  <node name="joy_node" pkg="joy" type="joy_node">
    <remap from="joy" to="joy_orig"/>
  </node>
  <!-- remap joy to emulate ps3joy mappings -->
  <node name="joy_remap" pkg="joy" type="joy_remap.py">
    <remap from="joy_in" to="joy_orig"/>
    <remap from="joy_out" to="joy"/>
    <rosparam command="load" file="$(find joy)/config/ps4joy.yaml"/>
  </node>
</launch>
